src/runtime/StaticStrings.cpp

```diff
@@ -94,6 +94,7 @@ void StaticStrings::initStaticStrings(AtomicStringMap* atomicStringMap)
     getSymbolSpecies.init(atomicStringMap, "get [Symbol.species]", strlen("get [Symbol.species]"));
     getSymbolSearch.init(atomicStringMap, "get [Symbol.search]", strlen("get [Symbol.search]"));
     getSymbolToStringTag.init(atomicStringMap, "get [Symbol.toStringTag]", strlen("get [Symbol.toStringTag]"));
+    getSymbolUnscopables.init(atomicStringMap, "get [Symbol.unscopables]", strlen("get [Symbol.unscopables]"));
     symbolSplit.init(atomicStringMap, "[Symbol.split]", strlen("[Symbol.split]"));
```

Comment:
`getSymbolUnscopables` seems unnecessary. Other looks fine to me.

Reply:
Yes. You are right. I fixed it.